tier2-icons.html

```diff
@@ -529,13 +529,17 @@
           <path fill-rule="evenodd" d="M23.8 4.2l-3.987-4a.676.676 0 0 0-.48-.2H1.867A1.889 1.889 0 0 0 0 1.907v20.187A1.889 1.889 0 0 0 1.867 24h20.267A1.889 1.889 0 0 0 24 22.093V4.667a.628.628 0 0 0-.2-.467zM4 7V5a1 1 0 0 1 1-1h10a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H5a1 1 0 0 1-1-1zm8 13a5 5 0 1 1 5-5 5 5 0 0 1-5 5z"/>
           <circle fill-rule="evenodd" cx="12" cy="15" r="3"/>
         </g>
+        <g id="scorm">
+          <path d="M20.9 4H3.1C1.4 4 0 5.4 0 7.1c0 .9.4 1.8 1.1 2.4v7.4C1.1 19.1 3 21 5.3 21h13.3c2.3 0 4.2-1.9 4.2-4.2V9.5C23.6 8.9 24 8 24 7.1 24 5.4 22.6 4 20.9 4zm0 12.8c0 1.2-1 2.2-2.2 2.2H5.3c-1.2 0-2.2-1-2.2-2.2v-6.6h17.8v6.6zm0-8.6H3.1C2.5 8.2 2 7.7 2 7.1 2 6.5 2.5 6 3.1 6h17.8c.6 0 1.1.5 1.1 1.1 0 .6-.5 1.1-1.1 1.1z"/>
+          <path d="M10 16.2H6c-.6 0-1-.4-1-1v-2c0-.6.4-1 1-1h4c.6 0 1 .4 1 1v2c0 .5-.4 1-1 1z"/>
+        </g>
         <g id="search">
           <path d="M23.12 18.88l-4.24-4.24c-.88-.88-2.29-1.08-3.4-.58l-1.16-1.16C15.37 11.55 16 9.85 16 8c0-4.42-3.58-8-8-8S0 3.58 0 8s3.58 8 8 8c1.85 0 3.55-.63 4.91-1.68l1.14 1.14c-.19.4-.29.84-.29 1.3 0 .8.31 1.55.88 2.12l4.24 4.24c.59.59 1.35.88 2.12.88s1.54-.29 2.12-.88c.57-.56.88-1.32.88-2.12s-.31-1.56-.88-2.12zM2 8c0-3.31 2.69-6 6-6s6 2.69 6 6-2.69 6-6 6-6-2.69-6-6zm19.71 13.71c-.39.39-1.02.39-1.42 0l-4.24-4.25c-.18-.18-.29-.43-.29-.7 0-.26.1-.49.27-.68l.06-.06c.18-.17.42-.26.67-.26.27 0 .52.1.71.29l4.24 4.24c.39.39.39 1.03 0 1.42z"/>
         </g>
         <g id="seating">
           <path d="M20 12h-3V9.54c.898-.153 1.783-.37 2.65-.65a2 2 0 0 0 1.07-2.92l-3-5a2.058 2.058 0 0 0-1.27-.92A2.3 2.3 0 0 0 16 0a1.926 1.926 0 0 0-1.06.31 6.142 6.142 0 0 1-2.87.69A7.148 7.148 0 0 1 8.89.21a2 2 0 0 0-2.61.76l-3 5A2.008 2.008 0 0 0 4.4 8.91c.854.264 1.722.48 2.6.65V12H4a2.006 2.006 0 0 0-2 2v2a2.006 2.006 0 0 0 2 2h1v4.7A1.3 1.3 0 0 0 6.3 24h.4A1.3 1.3 0 0 0 8 22.7V22h8v.69A1.315 1.315 0 0 0 17.31 24h.38A1.315 1.315 0 0 0 19 22.69V18h1a2.006 2.006 0 0 0 2-2v-2a2.006 2.006 0 0 0-2-2zM5 7l3-5c1.26.648 2.654.99 4.07 1A7.8 7.8 0 0 0 16 2l3 5a24.054 24.054 0 0 1-7.11 1.02A23.2 23.2 0 0 1 5 7zm10 2.84V12H9V9.86a26.752 26.752 0 0 0 6-.02zM16 20H8v-2h8v2zm4-4H4v-2h16v2z" fill-rule="evenodd"/>
         </g>
-        <g id="self-assessment">
+        <g id="self-assessment" mirror-in-rtl="true">
```

Comment:
These additional "mirror-in-rtl" entries were my fault -- I modified the SVGs a little while ago without re-running the build, so your build is picking them up. They're OK to go in, so no change on your part required. Sorry about that!